Content.Shared/Actions/ActionContainerSystem.cs

```diff
@@ -121,6 +143,8 @@ public override void Initialize()
 
         DebugTools.AssertEqual(action.Container, newContainer);
         DebugTools.AssertNull(action.AttachedEntity);
+        if (HasComp<MindComponent>(action.Container))
+            action.ItemIconStyle = ItemActionIconStyle.BigAction;
 
```

Comment:
Transferring actions shouldn't unexpectedly change properties of the action. If mind actions need a specific icon style, that should be configured on the action or maybe when the action is added to the mind, not when an action gets moved between containers.